[OV] Update optimization docs page with information about VLMs (#1007)
huggingface/optimum-intel: docs/source/openvino/export.mdx

                        and inference latency. Default value is 1.0.
  --sym                 Whether to apply symmetric quantization
  --group-size GROUP_SIZE
                        The group size to use for quantization. Recommended value is 128 and -1 uses per-column
                        quantization.
  --dataset DATASET     The dataset used for data-aware compression or quantization with NNCF. You can use the one
                        from the list ['wikitext2','c4','c4-new'] for language models or
                        ['conceptual_captions','laion/220k-GPT4Vision-captions-from-LIVIS','laion/filtered-wit'] for
                        diffusion models.
  --all-layers          Whether embeddings and last MatMul layers should be compressed to INT4. If not provided an
                        weight compression is applied, they are compressed to INT8.
  --awq                 Whether to apply AWQ algorithm. AWQ improves generation quality of INT4-compressed LLMs, but
                        requires additional time for tuning weights on a calibration dataset. To run AWQ, please also

>                         provide a dataset argument. Note: it's possible that there will be no matching patterns in the

Reviewer: ```suggestion
                        provide a dataset argument. Note: it is possible that there will be no matching patterns in the
```

                        model to apply AWQ, in such case it will be skipped.
  --scale-estimation    Indicates whether to apply a scale estimation algorithm that minimizes the L2 error between
                        the original and compressed layers. Providing a dataset is required to run scale estimation.
                        Please note, that applying scale estimation takes additional memory and time.
  --gptq                Indicates whether to apply GPTQ algorithm that optimizes compressed weights in a layer-wise
                        fashion to minimize the difference between activations of a compressed and original layer.
                        Please note, that applying GPTQ takes additional memory and time.
  --sensitivity-metric SENSITIVITY_METRIC

>                         The sensitivity metric for assigning quantization precision to layers. Can be one of the

Reviewer: ```suggestion
                        The sensitivity metric for assigning quantization precision to layers. It can be one of the
```

                        following: ['weight_quantization_error', 'hessian_input_activation',
                        'mean_activation_variance', 'max_activation_variance', 'mean_activation_magnitude'].
  --num-samples NUM_SAMPLES
                        The maximum number of samples to take from the dataset for quantization.
  --disable-stateful    Disable stateful converted models, stateless models will be generated instead. Stateful models
                        are produced by default when this key is not used. In stateful models all kv-cache inputs and
                        outputs are hidden in the model and are not exposed as model inputs and outputs. If --disable-
                        stateful option is used, it may result in sub-optimal inference performance. Use it when you
                        intentionally want to use a stateless model, for example, to be compatible with existing

>                         OpenVINO native inference code that expects kv-cache inputs and outputs in the model.

Reviewer: ```suggestion
                        OpenVINO native inference code that expects KV-cache inputs and outputs in the model.
```

  --disable-convert-tokenizer
                        Do not add converted tokenizer and detokenizer OpenVINO models.
```

You can also apply fp16, 8-bit or 4-bit weight-only quantization on the Linear, Convolutional and Embedding layers when exporting your model by setting `--weight-format` to respectively `fp16`, `int8` or `int4`.

Export with INT8 weights compression:
```bash
optimum-cli export openvino --model meta-llama/Meta-Llama-3-8B --weight-format int8 ov_model/
```

Export with INT4 weights compression:
```bash
optimum-cli export openvino --model meta-llama/Meta-Llama-3-8B --weight-format int4 ov_model/
```

> Export with INT4 weights compression and a data-aware AWQ algorithm:

Reviewer: ```suggestion
Export with INT4 weights compression and a data-aware AWQ and Scale Estimation algorithms:
```

```bash
optimum-cli export openvino --model meta-llama/Meta-Llama-3-8B \

>     --weight-format int4 --awq --dataset wikitext2 ov_model/

Reviewer: ```suggestion
    --weight-format int4 --awq --scale-estimation --dataset wikitext2 ov_model/
```
